gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -103,13 +105,14 @@ def writeToSheet(sheet_api_instance, assignment_scores, assignment_name = ASSIGN
                     }
                 }
             }
-            response = sheet_api_instance.batchUpdate(spreadsheetId=SPREADSHEET_ID, body=create_sheet_request).execute()
+            request = sheet_api_instance.batchUpdate(spreadsheetId=SPREADSHEET_ID, body=create_sheet_rest_request)
+            response = make_request(request)
             sheet_id = response['replies'][0]['addSheet']['properties']['sheetId']
         else:
             sheet_id = sub_sheet_titles_to_ids[assignment_name]
-        time.sleep(SLEEP_TIME)
         update_sheet_with_csv(assignment_scores, sheet_api_instance, sheet_id)
-        logger.info(f"Successfully updated spreadsheet with: {assignment_name}")
+        logger.info(f"Successfully updated spreadsheet with: {assignment_name} after {number_of_retries_needed_to_update_sheet} retries")
+        number_of_retries_needed_to_update_sheet = 0
     except HttpError as err:
         logger.error(err)
 
```

Comment:
```suggestion
except Exception as err:
    logger.error(f"An unknown error has occurred: {err}")
```

Reply:
Just in case we have an edge case where `HttpError` does not catch our error.

Reply:
Good idea, Mehul!